tests/integration_tests.rs

```diff
@@ -24,178 +25,231 @@ fn bat() -> Command {
 
 #[test]
 fn basic() {
-    bat()
+    let assert = bat()
         .arg("test.txt")
-        .assert()
+        .assert();
+    let stdout = String::from_utf8_lossy(&assert.get_output().stdout);
+    let stderr = String::from_utf8_lossy(&assert.get_output().stderr);
+    println!("stdout={:#?}", stdout);
+    println!("stderr={:#?}", stderr);
+    assert
```

Comment:
Could you please revert those changes? The tests already provides a (suboptimal?) debug output in case the tests fail. I'd rather not have these repetitive `println!` statements in here.

Reply:
Just the `println!`'s?

Reply:
Well, no. All of the modifications, except for the `predicate`-changes which are required to make the tests pass.

Is the normalization still required after the `.gitattributes` changes?

Reply:
Yes, for the echo's (if they work).